src/Workspaces/Remote/ServiceHub/Host/RemoteWorkspace.SolutionCreator.cs

```diff
@@ -412,71 +415,6 @@ private async Task<Solution> UpdateProjectAsync(Project project, ProjectStateChe
                 return project.Solution;
             }
 
-            private async Task<Project> UpdateProjectInfoAsync(Project project, Checksum infoChecksum, CancellationToken cancellationToken)
```

Comment:
oh.  is the idea that this logic moved into core logic the workspace jsut supports itself?

Reply:
yes